### PR TITLE
Add cloudwatch alarms docs

### PR DESCRIPTION
**Pull request by mohabusama**

### CODE REVIEW

File: docs/user/check-ref/cloudwatch_wrapper.rst [hunk 1]
```diff
@@ -53,3 +53,25 @@ The desired metric can now be queried in ZMON:
...
+
+    Retrieve cloudwatch alarms filtered by state value.
+
+    :param alarm_names: List of alarm names.
+    :type alarm_names: list
+
+    :param alarm_name_prefix: Prefix of alarms. Cannot be specified if ``alarm_names`` is specified.
+    :type alarm_name_prefix: str
+
+    :param state_value: State value used in alarm filtering. Available values are STATE_OK, STATE_ALARM(default) and STATE_DATA.
```
Comment (hjacobs) [inline docs/user/check-ref/cloudwatch_wrapper.rst]: this won't work for users as they have to use the string values, so best to mention the string values like "ALARM" here in the docs

File: docs/user/check-ref/cloudwatch_wrapper.rst [hunk 2]
```diff
@@ -53,3 +53,25 @@ The desired metric can now be queried in ZMON:
...
+    :type state_value: str
+
+    :param action_prefix: Action prefix.
```
Comment (hjacobs) [inline docs/user/check-ref/cloudwatch_wrapper.rst]: what's this exactly? maybe give an example and/or link to docs

File: docs/user/check-ref/cloudwatch_wrapper.rst [hunk 3]
```diff
@@ -53,3 +53,25 @@ The desired metric can now be queried in ZMON:
...
+    :type action_prefix: str
+
+    :param max_records: Maximum records to be returned. Default is 50.
+    :type max_records: int
+
+    :return: List of MetricAlarms.
```
Comment (hjacobs) [inline docs/user/check-ref/cloudwatch_wrapper.rst:78]: give an example return data structure

File: docs/user/check-ref/cloudwatch_wrapper.rst (hunk 1 above)
Comment (mohabusama) [inline docs/user/check-ref/cloudwatch_wrapper.rst]: Done.

File: docs/user/check-ref/cloudwatch_wrapper.rst (hunk 2 above)
Comment (mohabusama) [inline docs/user/check-ref/cloudwatch_wrapper.rst]: Added example and linked docs.

File: docs/user/check-ref/cloudwatch_wrapper.rst (hunk 3 above)
Comment (mohabusama) [inline docs/user/check-ref/cloudwatch_wrapper.rst:78]: Added example.

**Comment by hjacobs** [comment]
:+1:

**Comment by mohabusama** [comment]
👍

**Pull request merged**